src/invariant/EventsAreConsistentWithEntryDiffs.h

```diff
@@ -0,0 +1,39 @@
+#pragma once
+
+// Copyright 2025 Stellar Development Foundation and contributors. Licensed
+// under the Apache License, Version 2.0. See the COPYING file at the root
+// of this distribution or at http://www.apache.org/licenses/LICENSE-2.0
+
+#if defined(__SIZEOF_INT128__) || defined(_GLIBCXX_USE_INT128)
```

Comment:
This was a quick workaround to run the invariant locally and supercluster(which we haven't done yet). The invariant depends on the compiler to provide a int128 definition, and core will fail at runtime if you enable the invariant but the compiler does not support int128. I thought this would be fine because we just want to run the invariant to validate events, but it is a hack. That will mess with tests and test tx meta The alternative is to pull in a library that provides in128 support. We only need the + and - operators, so we can remove everything else from the library. That sounds like the better option.

Reply:
Another issue - if you try to enable the invariant with regex and don't have compiler support, then the invariant will be ignored instead of failing at runtime. This is enough of a reason to use a library.

Reply:
This was addressed in a recent commit that imports a library instead.